cpp/src/arrow/python/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@
 # arrow_python
 #
 
-find_package(PythonLibsNew REQUIRED)
+find_package(Python3Alt REQUIRED)
 find_package(NumPy REQUIRED)
```

Comment:
How about we find NumPy in `FindPython3Alt.cmake` and don't call `find_package(NumPy REQUIRED)` here?
Because `FindPython3.cmake` can find both Python3 and NumPy at once.  I think that `FindPython3Alt.cmake` should do so.

Reply:
Will try, thank you.